quix-frontend/client/src/lib/sql-autocomplete/adapter/sql-autocomplete-adapter.ts

```diff
@@ -48,6 +55,29 @@ export class SqlAutocompleter implements IAutocompleter {
     }
   }
 
+  public async getCompletionItemsFromQueryContextColumn(queryContext: QueryContext) {
+    return queryContext.contextType !== ContextType.Column ?
```

Comment:
use `===` it will be more readable
first you handle the big case and then the fallback

Reply:
actually.. in seceond though 
I think that swich case will be better here..

Reply:
like this?
switch (queryContext.contextType) {
  case ContextType.Column:
    return this.translateAndGetQueryContextColumns(queryContext.tables, queryContext.prefix);
  default:
    return [];
}